### DISCUSSION TITLE
What BMS should I get to test bluetooth?

### DISCUSSION THREAD

**Discussion by rossandrews** [jblance/mpp-solar#350] [Q&A]
I have some LiFePO4 cells laying around that need to be turned into a battery. I would like to make a 24V battery, which BMS should I use to work with the bluetooth connectivity of mpp-solar?

**Comment by jblance** [comment]
the current code is/was based on JKBMS (though a passive balancer, not the
newer/better active ones) and I am building a battery with Daly BMS which
have BLE / RS485 and CAN connections for further prototyping
